Wordle.Api/Wordle.Api.Tests/PlayerServiceTests.cs

```diff
@@ -5,40 +5,73 @@
 using System.Text;
 using System.Threading.Tasks;
 using Wordle.Api.Data;
+using Wordle.Api.Dtos;
 using Wordle.Api.Services;
 
-namespace Wordle.Api.Tests
+namespace Wordle.Api.Tests;
```

Comment:
You've got lots of whitespace/formatting changes making your git diff hard to read. Make sure you are only making changes relevant to the assignment.